src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -304,6 +304,27 @@ public static Colony getColony(@NotNull final World w, @NotNull final BlockPos p
     }
 
     /**
+     * Get all colonies in all worlds.
+     * @param abandonedSince time in hours since the last contact.
+     * @return a list of colonies.
+     */
+    @NotNull
+    public static List<Colony> getColonies(final int abandonedSince)
```

Comment:
I think you should rename this, because the method name itself doesn't give me any clue that its getting abandoned colonies.